SMDS_3/include/CGAL/tetrahedron_soup_to_triangulation_3.h

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) 2009-2014 INRIA Sophia-Antipolis (France).
+// Copyright (c) 2010-2013 GeometryFactory Sarl (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     :  Mael Rouxel-Labbé, Maxime Gimeno, Jane Tournois
+//
+//******************************************************************************
+// File Description :
+//******************************************************************************
+
+#ifndef CGAL_SMDS_3_TETRAHEDRON_SOUP_TO_C3T3_H
+#define CGAL_SMDS_3_TETRAHEDRON_SOUP_TO_C3T3_H
+
+#include <CGAL/license/SMDS_3.h>
+
+#include <CGAL/SMDS_3/tet_soup_to_c3t3.h>
+#include <CGAL/SMDS_3/internal/SMDS_3_helper.h>
+#include <CGAL/Named_function_parameters.h>
+#include <CGAL/boost/graph/named_params_helper.h>
+
+#include <vector>
+#include <array>
+#include <boost/unordered_map.hpp>
+
+namespace CGAL {
+
+  /** \ingroup PkgSMDS3Functions
+  * builds a 3D triangulation from a soup of tetrahedra.
+  *
+  * @tparam TetrahedronRange a model of `Range` whose value type is
+  * a `Tetrahedron_3`. The point type of the tetrahedra must be convertible
+  * to `Triangulation::Point`
+  * @tparam Triangulation a 3D triangulation class that has
+  * a vertex base model of `SimplicialMeshVertexBase_3`
+  * and a cell base model of `SimplicialMeshCellBase_3`
+  *
+  * @param tets the set of finite tetrahedra of a valid CGAL triangulation.
+  * Each element in the range is the geometric description of the
+  * corresponding cell in the triangulation.
+  *
+  * @returns the 3D triangulation built from \p tets
+  *
+  * @post the output triangulation must be a triangulation of the convex hull of `tets`
+  */
```

Comment:
+ add a link to the example?

Reply:
Isn't it automatic?

Reply:
Here in the "latest" [doc](https://doc.cgal.org/latest/Polygon_mesh_processing/group__PMP__repairing__grp.html#ga0dec58e8a0112791f72ebbe77bac074b) it seems to be added automatically (there is nothing in the header)
@sloriot are examples linked automatically?

Reply:
Then it should be fine :-)

Reply:
The documentation has still no link to the examples: https://cgal.github.io/5693/v11/SMDS_3/group__PkgSMDS3Functions.html#gad2f0b256c958cea9aff7fb4f855b53eb